src/bin/cargo/commands/check.rs

```diff
@@ -26,6 +26,7 @@ pub fn cli() -> App {
             "Check all targets",
         )
         .arg_release("Check artifacts in release mode, with optimizations")
+        .arg_force_rebuild("Force rebuild of this crate")
```

Comment:
What happens if you pass multiple crates to be checked in a workspace (`cargo check --all --force-rebuild`) or multiple targets in a lone crate (`cargo check --all-targets --force-rebuild`)?

Reply:
Thanks for bringing this up, I wasn't sure of what the wording should be here. Perhaps this should say `Force rebuild of this workspace`?

Reply:
"the selected target(s)" or "the selected package(s)" maybe?

Reply:
Selected targets probably works, implying package selection too.